### PR TITLE
Use literal block style for multi-line YAML

### PR DESCRIPTION
**Pull request by omus**
Allows for Julia code to not require the use of semi-colon which is matches how most users will write Julia code. An additional benefit is that it allows for use of single line comments.

### CODE REVIEW

**Comment by christopher-dG** [comment]
Thanks for the bump, I'll get to this on the weekend.

**Comment by omus** [comment]
> Also, the test files will need to be updated (you can include("test/runtests.jl") and accept the prompts to update the files).

It would be good to document this in a CONTRIBUTING.md file

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ julia:
 before_script:
   - git config --global user.name Tester
   - git config --global user.email te@st.er
-script: travis_wait julia --project -e 'using Pkg; Pkg.test(; coverage=true)'
+script: travis_wait julia --project -e 'using Pkg; Pkg.test(coverage=true)'
```
Comment (christopher-dG) [inline .travis.yml]: I generally like using semicolons for all keyword arguments but these changes are fine with me.

Comment (omus) [reply .travis.yml]: I also tend to use semicolons. In this particular case it seems worthwhile to be a bit more terse

**Pull request merged**